src/views/PageView.vue

```diff
@@ -17,14 +16,143 @@ const userStore = useUserStore()
 </script>
 
 <template>
-  <button class="google-button" @click="userStore.logout()">
-    <img src="../assets/google-logo.png" alt="Google logo">
-    <span>Se déconnecter</span>
-  </button>
-  <button class="button-text" @click="router.push('/modules')">Modules</button>
-  <button class="button-text" @click="router.push('/tips')">Tips</button>
-  <ModulesView v-if="location.path == '/modules'" />
-  <TipsView v-if="location.path == '/tips'" />
+  <main id="page-view">
+    <section>
+      <img src="../assets/logo.png" alt="Logo">
+      <nav>
+        <RouterLink to="/modules">Modules</RouterLink>
+        <RouterLink to="/tips">Astuces</RouterLink>
+        <RouterLink to="/accounts" v-if="userStore.isAdmin">Comptes</RouterLink>
+      </nav>
+    </section>
+    <section>
+      <header>
+        <h1 class="title-text">UDSP59 FORMATION</h1>
+        <button class="disconnect-button" @click="userStore.logout()" title="Se déconnecter">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+            <path fill="#BA1626"
+              d="M502.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L402.7 224 192 224c-17.7 0-32 14.3-32 32s14.3 32 32 32l210.7 0-73.4 73.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0l128-128zM160 96c17.7 0 32-14.3 32-32s-14.3-32-32-32L96 32C43 32 0 75 0 128L0 384c0 53 43 96 96 96l64 0c17.7 0 32-14.3 32-32s-14.3-32-32-32l-64 0c-17.7 0-32-14.3-32-32l0-256c0-17.7 14.3-32 32-32l64 0z" />
+          </svg>
+        </button>
+      </header>
+
+      <button class="add-button" @click="" title="Ajouter">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+          <path fill="#FFFFFF"
+            d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z" />
+        </svg>
+      </button>
+      <ModulesView v-if="location.path == '/modules'" />
+      <TipsView v-if="location.path == '/tips'" />
+      <AccountsView v-if="location.path == '/accounts'" />
+    </section>
+  </main>
 </template>
 
-<style></style>
\ No newline at end of file
+<style>
+main#page-view {
```

Comment:
Avoid using ID for style, could create issue with specificity.
Prefer class usage